test/integration/docker_build_test.exs

```diff
@@ -104,6 +104,26 @@ defmodule ExDockerBuild.Integration.DockerBuildTest do
       assert log =~ "STEP 5/6 : VOLUME vol_storage:/myvol"
       assert log =~ "STEP 6/6 : CMD [\"cat\", \"/myvol/greeting\"]"
     end
+
+    test "create and manage persistent storage as volumes that can be attached to containers" do
+      volume_name = "vol_storage"
+      instructions = [
+        {"FROM", "alpine:3.8"},
+        {"VOLUME", volume_name},
+        {"VOLUME", "vol_storage:/myvol"}
+      ]
+
+      _log =
```

Comment:
if you are not using the logs you can use `@tag capture_log :true` flag before the test definition